cypress/integration/datepicker_page_spec.ts

```diff
@@ -155,10 +155,10 @@ describe('Datepicker demo page test suite', () => {
       datepicker.isDatepickerNavigationFullyActiveAndCorrect('date', 'body', datepicker.monthNames[4], '2015');
     });
 
-    it('when user clears input and add there date in bad format, click "Enter" - "Invalid date" shown', () => {
-      datepicker.clearInputAndSendKeys(basic, '20,10,2015', 0);
```

Comment:
changes to this file doesn't seem to be appropriate, please revert and express what you are planning to test here

Reply:
Hi, I just modified the test case to work based on the fix to this issue #5576 . 
When invalid date is entered, it is retained and not passed on to reflect on the datepicker.